modules/olm-installing-opm.adoc

```diff
@@ -15,7 +15,7 @@ You can install the `opm` CLI tool on your Linux, macOS, or Windows workstation.
 
 .Procedure
 
-. Navigate to the link:https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest/[OpenShift mirror site] and download the latest version of the tarball that matches your operating system.
+. Navigate to the link:https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.7.0/[OpenShift mirror site] and download the latest version of the tarball that matches your operating system.
```

Comment:
https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.7.0/ appears to have the latest 4.7 build that I'm finding right now. The `opm version` for that spits out `v1.15`.

https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest/ is still pointing to the older `v1.14` build currently. Ideally, I want to update the URL to point to a `latest-4.7/` directory, but that hasn't been created yet. I'll update this again when that has happened.

Reply:
Looks like `latest/` has been updated to point to the 4.7.0 builds, so I'll revert this here.

Reply:
Actually, `latest-4.7/` exists now, so I'll use that! :tada: